engine/tests/phpunit/Elgg/lib/entities/EntityTypeRegistrationTest.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Elgg\lib\entities;
+
+/**
+ * @group Entities
+ */
+class EntityTypeRegistrationTest extends \Elgg\TestCase {
+
+	public function setUp() {
+		$mocks = new \Elgg\Tests\EntityMocks($this);
+		unset(_elgg_services()->entityTypes); // previous state is floating over
```

Comment:
@mrclay This stuff is bugging me. Maybe we should go back to your original proposal of adding a method to clear/backup `_elgg_services()`. Test cases should run independently from each other, and this stuff keeps popping up and hard to figure out why.